compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -3013,6 +3013,28 @@ public void visit(BLangTypeInit cIExpr) {
                 if (!cIExpr.initInvocation.argExprs.isEmpty()) {
                     BLangExpression iteratorExpr = cIExpr.initInvocation.argExprs.get(0);
                     BType constructType = checkExpr(iteratorExpr, env, symTable.noType);
+                    BType closeableIteratorType = symTable.langQueryModuleSymbol.scope
+                            .lookup(names.fromString(BLangCompilerConstants.ABSTRACT_CLOSEABLE_ITERATOR_CLASS))
```

Comment:
Should we add a `Name` in `org.wso2.ballerinalang.compiler.util.Names` instead?

Reply:
fixed in: bf7f95d592214df4df84a89e0b02cee38e7cb162